nextcord/interactions.py

```diff
@@ -204,23 +204,10 @@ def _from_data(self, data: InteractionPayload):
             self.message = None
 
         self.user: Optional[Union[User, Member]] = None
-        self._permissions: int = 0
+        self._app_permissions: int = 0
 
-        # TODO: there's a potential data loss here
         if self.guild_id:
-            guild = self.guild or Object(id=self.guild_id)
-            try:
-                member = data["member"]
-            except KeyError:
-                pass
-            else:
-                self.user = Member(state=self._state, guild=guild, data=member)  # type: ignore
-                self._permissions = int(member.get("permissions", 0))
-        else:
-            try:
-                self.user = User(state=self._state, data=data["user"])
-            except KeyError:
-                pass
+            self._app_permissions = int(data.get("app_permissions"))  # type: ignore
```

Comment:
I think self.guild check can be removed as well and just have
`data.get("app_permissions", 0)`

The property could also stay as `_permissions` I think to reduce the PR noise (fewer changes)